playbooks/common_playbook.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: raspberry common config
+  hosts: raspberry
+
+  pre_tasks:
+    - name: install python3
+      raw: test -r /usr/bin/python3 || (apt-get update && apt-get install -y python3)
+      tags: always
+
+    - name: install pip3 and python3-venv
+      apt:
+        name:
+          - python3-pip
+          - python3-venv
+        state: latest
+        update_cache: true
+      become: true
+      register: apt_output
+      tags: always
+
+    - name: full upgrade of all packages
+      apt:
+        upgrade: full
+      become: true
+      when: apt_output.changed
+      tags: dangerous
+
+    - name: reboot of RPI
+      reboot:
+      when: apt_output.changed
+      become: true
+      tags: dangerous
+
+  roles:
+    - role: geerlingguy.docker_arm
```

Comment:
Do not we need a step/role to install geerlingguy.docker_arm ?  (a command like ansible-galaxy install eerlingguy.docker_arm ?)

Reply:
If you install the `requirements.yml` it should work

Reply:
the command is `ansible-galaxy install requirements.yml` I should've added this step in the README.md !